agent/bench-scripts/postprocess/uperf-postprocess

```diff
@@ -117,23 +113,27 @@ if (opendir($dh, $tool_group_dir)) {
 		}
 	}
 } else {
-	print "$script: could not find any directories in $tool_group_dir which matched $tool_label_pattern\n";
+	print STDERR "$script: could not find any directories in $tool_group_dir which matched $tool_label_pattern\n";
 }
 closedir  $dh;
 foreach $this_tool_dir (sort keys %tool_ids) {
-	my $res;
 	my $this_tool_id = $tool_ids{$this_tool_dir};
 	my $role = "host";
-	my $url;
 	my $hostname;
 	($role, $hostname) = split(/:/,$this_tool_id);
-	my @cpubusy_samples;
-	$res = get_cpubusy_series($this_tool_dir, \@cpubusy_samples );
+	my %cpubusy_samples;
+	my $res = get_cpubusy_series($this_tool_dir, \%cpubusy_samples );
 	if ( $res == 0 ) {
 		my %this_cpubusy_dataset;
 		my $description = "Total busy CPU usage, averaged over N seconds, where 1.0 = 1 logical CPU";
-		my $mean = get_mean(\@cpubusy_samples);
-		%this_cpubusy_dataset = ( get_label('uid_label') => create_uid('hostname_label'), get_label('description_label') => $description, get_label('role_label') => $role, get_label('hostname_label') => $hostname, get_label('value_label') => $mean, get_label('timeseries_label') => \@cpubusy_samples );
+		%this_cpubusy_dataset = (
```

Comment:
Did you want to do here what you did in `linpack-postprocess` and push the value directly without storing it first?

Reply:
Sure....